app/chat_endpoint_handlers.py

```
@@ -298,6 +305,78 @@ async def chat_message_internal(chat_id: str, user_id: str, action: Action, back
         )
     return {"message": response}
 
+async def chat_image_internal(chat_id: str, user_id: str, make_image: MakeImage, background_tasks: BackgroundTasks):
```

Ruff (0.12.2)

Unused function argument: `make_image`

(ARG001)